src/funding/status-card/status-card.scss

```diff
@@ -0,0 +1,66 @@
+@import "src/resources/elements/primeDesignSystem/colors";
+pcard {
```

Comment:
Will this not overwrite pcard styles?

Maybe a selector for `<pcard>` here safer